app/src/components/common/guestInvite.tsx

```diff
@@ -0,0 +1,152 @@
+import React, {useState} from 'react';
+import {object, string} from 'yup';
+import {useFormik} from 'formik';
+import {
+  Box,
+  Button,
+  TextField,
+  FormHelperText,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@mui/material';
+import {createTheme, ThemeProvider} from '@mui/material/styles';
+import ControlPointIcon from '@mui/icons-material/ControlPoint';
+import {useInviteGuestMutation} from '../../services/coordinator';
+
+const theme = createTheme({
```

Comment:
We shouldn't have to create a theme and theme provider just for styling this button. You can sx or styled components to make these changes.

Reply:
To add a little more context. Looks like this was done to add the primary color from Figma to the buttons? We still haven't gotten a definite answer on whether this is the color we are going with so don't worry about if for now. When we get the okay, we'll update the color in the main theme and the changes should be applied. It would probably be good to check with design over the next week.